src/AnnotatedCommand.php

```diff
@@ -225,8 +225,9 @@ protected function setCommandArguments($commandInfo)
      */
     protected function checkUsesInputInterface($params)
     {
+        /** @var \ReflectionParameter $firstParam */
         $firstParam = reset($params);
-        return $firstParam instanceof InputInterface;
+        return $firstParam->getClass() && $firstParam->getClass()->implementsInterface(InputInterface::class);
```

Comment:
Ah, I see. Thanks for catching that. Looks like a cut-copy-paste error. This feature did work once in the past, really. 😉 

I think I misdiagnosed the problem. I am guessing that `firstParam` is a boolean `false` if `reset($params)` is called when `$params` is empty. Please confirm.